tools/github/Add-AzsdkMilestones.ps1

```diff
@@ -52,7 +60,8 @@ $milestones = do {
         DueOn = $next.ToString('s') + 'Z'
     }
 
-    $date = $next
+    # The next date to consider is the first of the following month.
+    $date = [DateTimeOffset]::Parse("$($next.AddMonths(1).ToString('yyyy-MM'))-01T23:59:59Z")
```

Comment:
Seems pointless to do this - just use `$date = $date.AddMonths(1)` since the first line in the loop will parse this format again anyway.

Reply:
Uh... yeah.   I overlooked that the adjustment happened each tick.  My bad.